internal/config/main_config.go

```diff
@@ -60,7 +60,8 @@ var config struct {
 			URL string
 		}
 		Sources struct {
-			URL string
+			URL   string
+			AppId string
```

Comment:
Should we define a static value for this in devel and test atleast?

Reply:
I think it needs to be configurable for devel mode, because we are using sources locally